plugin.php

```diff
@@ -1,4 +1,4 @@
-<?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
```

Comment:
Revert this change, since `WordPress.Files.FileName.InvalidClassFileName` shouldn't be excluded, per the comment above.

Reply:
Not sure I understand. The inline `phpcs:ignore` is no longer necessary as it's in the ruleset.